templates/plugin/salt_master.conf.erb

```diff
@@ -0,0 +1,58 @@
+# /etc/salt/master.d/foreman.config Example configuration
+#
+# This file summarizes configurations for the salt-master. Modify directories and
+# parameters to fit your setup. When you're done, remove the .example from the
+# filename so the salt-master will make use of it.
+# Have a look at the [Foreman Salt Plugin Documentation](https://theforeman.org/plugins/foreman_salt/) for detailed explanations.
+#
+# After editing this file, run the following command to active the changes:
+# $ systemctl restart salt-master
+
+
+##
+# Autosign
+autosign_grains_dir: <%= scope.lookupvar('::foreman_proxy::plugin::salt::autosign_grains_dir') %>
```

Comment:
You no longer need to prefix it with `::` (same is true in the rest of the file):
```suggestion
autosign_grains_dir: <%= scope.lookupvar('foreman_proxy::plugin::salt::autosign_grains_dir') %>
```